src/store/ChartState.js

```diff
@@ -319,15 +323,16 @@ class ChartState {
         }
     }
 
-    scrollChartToLeft = () => {
+    scrollChartToLeft(epoch) {
```

Comment:
@shayanbinary the function should be defined as an arrow function because it's used as a callback function on [line 124](https://github.com/binary-com/SmartCharts/pull/731/files#diff-01fca571a632d65a132cb788015d68f1R124)